cdap-app-fabric/src/main/java/co/cask/cdap/scheduler/ScheduleNotificationSubscriberService.java

```diff
@@ -140,6 +143,13 @@ public void processNotification(DatasetContext context, Notification notificatio
       }
       jobQueue.addNotification(record, notification);
     }
+
+    @Override
+    public JobQueueDataset getJobQueue() {
```

Comment:
Same comment as above - before this refactoring, we used the instance variable directly so we didn't fetch it every time. If I didn't do this, then we would be calling Schedulers.getJobQueue every time - is that okay?